packages/fxa-settings/src/lib/channels/firefox.ts

```diff
@@ -251,8 +251,38 @@ export class Firefox extends EventTarget {
     this.send(FirefoxCommand.FxAStatus, options);
   }
 
-  fxaLogin(options: FxALoginRequest) {
+  async fxaLogin(options: FxALoginRequest): Promise<void> {
     this.send(FirefoxCommand.Login, options);
+
+    // In Playwright, we need to wait for the browser to send a web channel message
```

Comment:
I suspect this is probably a general issue with Playwright and webchannels. I think it is fine to land this as is and see if we have other webchannel timing issues.